spec/views/spree/admin/orders/edit.html.haml_spec.rb

```diff
@@ -54,7 +59,6 @@ def current_ability
       it "doesn't display a table of out of stock line items" do
         render
 
-        expect(rendered).not_to have_content "Out of Stock"
```

Comment:
What is the cause of this? Do we need to make the selector more specific (eg put inside a `within` block)?

Also below.